dev/benchmarks/macrobenchmarks/README.md

```diff
@@ -65,7 +65,7 @@ Choose one of the two benchmark types:
 
 * A "raw benchmark" records performance metrics from direct interactions with
   `dart:ui` with no framework. This kind of benchmark is good for benchmarking
-  low-level engine primitives, such as layer, picture, and semantics performance.
+  low-level engine primitives, such as a layer, picture, and semantics performance.
```

Comment:
this is actually referring to "layer performance", so I believe the original is preferable.

```suggestion
  low-level engine primitives, such as layer, picture, and semantics performance.
```

Reply:
Ah okay, tbh I was confused here so I will change it.